frontend/views/containers/group-settings/InvitationsTable.vue

```diff
@@ -189,39 +233,41 @@ export default {
         invitee: isAnyoneLink ? L('Anyone') : invitee,
         inviteSecret,
         inviteLink: buildInvitationUrl(this.currentGroupId, inviteSecret),
+        description: this.inviteStatusDescription({
+          isAnyoneLink, isInviteExpired, isInviteRevoked, isAllInviteUsed, quantity, numberOfResponses
+        }),
+        expiryInfo: isInviteExpired ? L('Expired') : isInviteRevoked ? L('Revoked') : isAllInviteUsed ? '' : this.readableExpiryInfo(expiryTime),
         status: {
           isExpired: isInviteExpired,
-          isActive: !isInviteExpired && !isAllInviteUsed,
-          description: this.inviteStatusDescription({
-            isAnyoneLink, isInviteExpired, isAllInviteUsed, quantity, numberOfResponses
-          }),
-          expiryInfo: isAllInviteUsed ? '' : this.readableExpiryInfo(expiryTime)
+          isActive: !isInviteExpired && !isInviteRevoked && !isAllInviteUsed,
+          isRevoked: isInviteRevoked
         }
       }
     },
     handleInviteClick (e) {
       if (e.target.classList.contains('js-btnInvite')) {
         sbp('okTurtles.events/emit', OPEN_MODAL, 'AddMembers')
       }
-    }
-  },
-  computed: {
-    ...mapGetters(['currentGroupState', 'ourUsername']),
-    ...mapState(['currentGroupId']),
-    invitesToShow () {
-      const { invites } = this.currentGroupState
-
-      if (!invites) { return [] }
-
-      const invitesList = Object.values(invites)
-        .filter(invite => invite.creator === INVITE_INITIAL_CREATOR || invite.creator === this.ourUsername)
-        .map(this.mapInvite)
-      const options = {
-        Active: () => invitesList.filter(invite => invite.status.isActive),
-        All: () => invitesList
+    },
+    async handleRevokeClick (inviteSecret) {
+      if (!confirm(L('Are you sure you want to revoke this link? This action cannot be undone.'))) {
+        return null
       }
 
-      return options[this.ephemeral.selectbox.selectedOption]()
+      try {
+        this.ephemeral.inviteRevokedNow = inviteSecret
+        const message = await sbp('gi.contracts/group/inviteRevoke/create', {
+          inviteSecret
+        }, this.currentGroupId)
+        await sbp('backend/publishLogEntry', message)
+        setTimeout(() => {
+          this.ephemeral.inviteRevokedNow = null
+        }, 2000)
+      } catch (e) {
+        this.ephemeral.inviteRevokedNow = null
```

Comment:
Does this also hide the invite-link entry? If so, does it make sense to keep it in the list so the user can try again after the issue is resolved, without having to refresh (at which point it will be there again)?

Reply:
No, it does not because when failed, the invite won't turn `revoked`. So the `inviteRevokeNow` does not have any effect. 

Here's the condition again, the last comparison guarantees that.

```
Active: () => invitesList.filter(invite => invite.status.isActive || (invite.status.isRevoked && invite.inviteSecret === this.ephemeral.inviteRevokedNow)),
```